docs/release-notes/v13.0.3.md

```diff
@@ -40,6 +40,18 @@ This feedback will then be gathered and analyzed to improve the overall performa
 Once you have configured the Runtime Monitoring system for your Project, you can now define the scraping rules for each of your Microservice containers. You can perform such action in the `Monitoring` tab of each Microservice's details page.  
 For more information, visit the related [documentation page](/development_suite/api-console/api-design/microservice-monitoring.md).
 
+### Whitelabeling for Self-Hosted Clients
+
+During Self-Hosted installations, it is now possible to customize certain product properties to suit client' needs. 
+
+The customizable properties are:
+- Login page logo;
+- Background of the login page;
+- Favicon page;
+- Sidebar logo (extended and collapsed).
+
+For more information, visit the related [documentation page] (/infrastructure/self-hosted/installation-chart/helm-values/whitelabeling.md).

```

Comment:
```suggestion
For more information, visit the related [documentation page] (/infrastructure/self-hosted/installation-chart/helm-values/whitelabeling.md).
```suggestion
For more information, visit the related [documentation page](/infrastructure/self-hosted/installation-chart/helm-values/whitelabeling.md).

```
```